core/linker/default.h

```diff
@@ -69,7 +77,7 @@ MEMORY
   FLASH_S  (r x) : ORIGIN = (FLASH_ORIGIN + SECURE_ALIAS_OFFSET + FLASH_OFFSET),
                    LENGTH = UVISOR_FLASH_LENGTH_MAX
   RAM_S    (rwx) : ORIGIN = (SRAM_ORIGIN + SECURE_ALIAS_OFFSET + SRAM_OFFSET),
-                   LENGTH = UVISOR_SRAM_LENGTH_USED - (STACK_SIZE + STACK_GUARD_BAND) - (STACK_SIZE_NP ? (STACK_SIZE_NP + STACK_GUARD_BAND) : 0)
+                   LENGTH = UVISOR_SRAM_LENGTH_USED - (STACK_SIZE + STACK_GUARD_BAND + STACK_GUARD_BAND) - (STACK_SIZE_NP ? (STACK_SIZE_NP + STACK_GUARD_BAND) : 0)
```

Comment:
This line is a change to v8-M, since guard band is now counted twice.

To make this easier to read, create the length for v8 and v7 separately. v7 doesn't care about `STACK_SIZE_NP`.

Reply:
Shouldn't it being counted twice?
Before the change we had one guard which was counted also for v8-M.

Reply:
What we can do now is:
For v7-M: Use 2 guards of 512 bytes.
For v8-M: In order to keep unchanged, keep the 32 bytes guard (although not used) and remove it when dealing with v8-M.
Is that acceptable?

Reply:
Sounds good